Jenkinsfile_CNP

```diff
@@ -0,0 +1,37 @@
+#!groovy
+
+@Library("Infrastructure")
+
+def type = "java"
+def product = "recipes"
+def component = "backend"
+
+def apiTestSecrets = [ 'plumsi-${env}' :
```

Comment:
```suggestion
def apiTestSecrets = [ 'plumsi-${env}' :
```

Bothered about `plumsi` here?

Reply:
Discussed with Thomas, this is probably going to change if we merge everything into a single shared infra repo with a generic name.